src/transformers/models/xlm_roberta_xl/configuration_xlm_roberta_xl.py

```diff
@@ -0,0 +1,154 @@
+# coding=utf-8
+# Copyright 2018 The Google AI Language Team Authors and The HuggingFace Inc. team.
```

Comment:
change docstring to Hugging Face and correct year

Reply:
changed as your suggestions